### PR TITLE
make accessing the Ember Global lazy

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

**Comment by rwjblue** [comment]
Test failures are from a parse error, not sure why yet

**Comment by Turbo87** [comment]
mmh... *why* are we doing this again?

**Comment by stefanpenner** [comment]
> mmh... why are we doing this again?

currently ember-cli-shims is required to be ordered specifically, although it is an AMD module. I would like to make its order no longer a requirement.

Are there concerns with this approach?

File: vendor/ember-cli-shims/app-shims.js [hunk 1]
```diff
@@ -211,7 +211,7 @@
     ];
     for (var i = 0, l = computedMacros.length; i < l; i++) {
       var key = computedMacros[i];
-      shims['ember-computed'][key] = Ember.computed[key];
+      shims['ember-computed'][key] = 'Ember.computed' + '.'+ key;
```
Comment (trentmwillis) [inline vendor/ember-cli-shims/app-shims.js]: What about just `Ember.computed` on line 201?

Comment (stefanpenner) [inline vendor/ember-cli-shims/app-shims.js]: 👍

**Comment by stefanpenner** [comment]
@Turbo87 you above comment leads me to believe you have concerns, is that the case? Please share, I don't want to 🚢  this if you have concerns.

**Comment by Turbo87** [comment]
> you above comment leads me to believe you have concerns, is that the case?

nope, I'm just having a hard time understanding what the advantages of this refactoring are since it seems to increase filesize and complexity 🤔

**Comment by stefanpenner** [comment]
@Turbo87 are you ok with it?

**Comment by Turbo87** [comment]
@stefanpenner now that the new module imports via `ember-cli-babel@6.6.0` are a thing I'm not sure there is much value anymore in improving this essentially deprecated addon 🤔

**Comment by Turbo87** [comment]
closing due to inactivity and deprecation of the addon

**Pull request closed without merging**